pyproject.toml

```diff
@@ -109,3 +111,47 @@ exclude_lines = [
     # Ignore typing overloads
     "@overload",
 ]
+
+[tool.tox]
+env_list = ["clean", "3.10", "3.11", "3.12", "3.13", "3.14", "build_docs"]
+requires = ["tox>=4.0", "tox-uv>=1.20"]
```

Comment:
I think this is subject to change if we decide to go with uv or not or something else